<!--Launches the action server and PID nodes-->
<launch>
    <group ns="control">
        <node name="WResponseControl_ArmMotors" pkg="wresponsecontrol" type="WResponseControl">
            <rosparam file="$(find wr_control_drive_arm)/config/arm_motor_PID_real.yaml" />
        </node>
        <node name="ArmControlActionServer" pkg="wr_control_drive_arm"
            type="ArmControlActionServer"
            output="screen">
            <rosparam
                file="$(find wr_control_drive_arm)/config/encoder_parameters_real.yaml"
                if="$(eval
        env('WROVER_HW') == 'REAL')" />
            <rosparam
                file="$(find
        wr_control_drive_arm)/config/encoder_parameters_mock.yaml"
                if="$(eval
        env('WROVER_HW') == 'MOCK')" />
        </node>
    </group>
</launch>